### PR TITLE
round the < to ( to make markdown work for meta.yml

### PR DESCRIPTION
**Pull request by lassefolkersen**
## PR checklist

Closes #XXX 
This is a minor change to make sure the markdown in the `meta.yml` renders correctly. Thanks for noticing @mashehu

### CODE REVIEW

File: modules/ascat/meta.yml [hunk 1]
```diff
@@ -22,16 +22,16 @@ input:
...
-            "gc_files": <optional>,
-            "minCounts": <optional>,
-            "chrom_names": <optional>,
-            "min_base_qual": <optional>,
-            "min_map_qual": <optional>,
-            "ref_fasta": <optional>,
-            "skip_allele_counting_tumour": <optional>,
-            "skip_allele_counting_normal": <optional>
+            "purity": (optional),
+            "ploidy": (optional>,
```
Comment (mashehu) [inline modules/ascat/meta.yml:26]:
```suggestion
            "ploidy": (optional),
```

**Review by mashehu — COMMENTED** [review]
There is also an <optional> in line 18.

File: modules/ascat/meta.yml (hunk 1 above)
Comment (lassefolkersen) [reply modules/ascat/meta.yml]: Ok, thanks I've corrected

**Review by mashehu — APPROVED** [review]

**Comment by lassefolkersen** [comment]
Ok, here's a new commit with _all_ the < converted to (, thanks @mashehu - also had to do some other changes because the linting started failing for empty files (it didn't before); which in turn needed an update in the stub-running documentation (being [pushed](https://github.com/nf-core/nf-co.re/pull/1079) right now).

**Pull request merged**